packages/ckb-indexer/src/collector.ts

```diff
@@ -1,21 +1,18 @@
-import {
-  Script,
-  utils,
-  Cell,
-  BaseCellCollector,
-  ScriptWrapper,
-} from "@ckb-lumos/base";
+import { utils, Cell, BaseCellCollector } from "@ckb-lumos/base";
 import { validators } from "ckb-js-toolkit";
 import {
   AdditionalOptions,
```

Comment:
it seems that `AdditionalSearchKey` is not a good type name, we can rename it to `SearchKeyFilter`

Reply:
refactor(ckb-indexer): rename